stdlib/database/jdbc2/src/main/ballerina/src/java.jdbc/client.bal

```diff
@@ -36,10 +36,41 @@ public type Client client object {
         return createClient(self, clientConf, sql:getGlobalConnectionPool());
     }
 
+    # Executes the sql query provided by the user, and returns the result as stream.
+    #
+    # + sqlQuery - The query which needs to be executed
+    # + rowType - The `typedesc` of the record that should be returned as a result. If this is not provided the default
+    #             column names of the query result set be used for the record attributes
+    # + return - Stream of records in the type of `rowType`
+    public remote function query(@untainted string sqlQuery, typedesc<record {}>? rowType = ())
+    returns stream<record{}, sql:Error> {
```

Comment:
Need indentation here. This is an issue with formatter which was fixed with https://github.com/ballerina-platform/ballerina-lang/pull/21354

Reply:
I took an update and ran the formatting tool yesterday (after the update from master) and again today and this is how it formatted. If I have the indentation here it reverts and puts back as above.

Reply:
AFAIK formatting tool with his fix is not released yet. @NipunaMarcus Any thoughts?

Reply:
We can leave this and fix them later, in case we need to merge this soon for the release :)

Reply:
Yes, +1. Once the formatter tool is fixed and released, I'll rerun the tool and fix these with the separate PR.